extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -141,7 +156,7 @@ interface HibernateOrmConfigPersistenceUnitDatabaseGeneration {
         @WithParentName
```

Comment:
since it's `generation.strategy` in the properties files, do we need to keep the `@WithParentName` ?